index.html

```diff
@@ -36,13 +34,160 @@ <h2>
+    <li class="bootstrap">bootstrap</li>
+    <li class="Ruby">Ruby</li>
+</ul>
+<button id="project-btn">See Project
+</button>
+</div>
+</div>
+<div id="placehokder-img-div">
+    <div class="placehokder-img">
+<h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+<p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul class="lang-skills">
+    <li class="html">html</li>
+    <li class="bootstrap">bootstrap</li>
+    <li class="Ruby">Ruby</li>
+</ul>
+<button id="project-btn">See Project
+</button>
+</div>
+</div>
+<div id="placehokder-img-div">
+    <div class="placehokder-img">
+<h1 id="placeholder-img-head">Profesional Art Printing Data</h1>
+<p id="palceholder-img-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul class="lang-skills">
+    <li class="html">html</li>
+    <li class="bootstrap">bootstrap</li>
+    <li class="Ruby">Ruby</li>
+</ul>
+<button id="project-btn">See Project
+</button>
+</div>
+</div>
+</section>
+
+<section id="about-me">
+    <h1 id="about-me-head">About me</h1>
+    <p id="about-me-para">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <button id="get-resume">Get My Resume
+    </button>
+    </section>
+    <section id="lang-section">
+    <img src="images/desktop-about-me-frame.svg" id="about-me-frame-desktop">
+    <div id="border"></div>
+    
+    <div id="language">
+   
+    <div id="shape"><span id="diamond-shape"> </span> <h2 id="languages">
+        Language
+
+    </h2></div>
+    <ul id="language-list">
+        <li class="language-list-js">Javascript</li>
+        <li class="language-list-Ruby">Ruby</li>
+        <li class="language-list-html">Html</li>
+        <li class="language-list-css">Css</li>
+    </ul>
+<div>
+    <div id="shape"><span id="square-shape"></span><h2 class="frameworks">
+        Frameworks
+
+    </h2>
+</div>
+    <ul id="language-list">
+        <li class="language-list-Bootstrap">Bootstrap</li>
+        <li class="language-list-ruby-on-rails">Ruby on Rails</li>
+        <li class="language-list-RSPec">RSPec</li>
+        <li class="language-list-CapyBara">CapyBara</li>
+        <li class="language-list-Selenium">Selenium</li>
+    </ul>
+</div>
+<div>
+    <div id="shape"><span id="circle-shape"></span><h2 id="skills">
+        Skills
+
+    </h2>
+</div>
+    <ul id="language-list">
+        <li class="language-list-Codekit">Codekit</li>
+        <li class="language-list-Github">Github</li>
+        <li class="language-list-Codepen">Codepen</li>
+        <li class="language-list-Gitlab">Gitlab</li>
+        <li class="language-list-Terminal">Terminal</li>
+    </ul>

```

Comment:
- [ ] to be semantically correct the  technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li` , kindly fix it 🙏